Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.16-alpine3.13 AS gobuilder
+FROM golang:1.17-alpine3.13 AS gobuilder
```

Comment:
Thank you :) Why did you pick alpine3.13 and not 3.14 here?

Reply:
Ah, I didn't want to introduce something that might have caused issues with any other parts of the project. I'm not sure if you were using anything 3.13 specific as that was how the tag was set. If not perhaps it would be better to choose the `golang:1.17-alpine` tag instead, since that will always target the latest alpine which is currently at 3.14, but will keep you on a desired golang version.

https://hub.docker.com/_/golang?tab=tags&page=1&ordering=last_updated&name=alpine

Let me know anyways, making the change is easy :)

Reply:
@zerok I've bumped the alpine related dependencies up to all be 3.14.

Reply:
Thank you :D Sorry, didn't have the time to get back to you yesterday.